home/views.py

```diff
@@ -46,4 +49,71 @@ def students(request):
             students_add_form.save()
             return redirect(reverse('students'))
         else:
-            return HttpResponseBadRequest('Некорректно заполнены в форме')
+            return HttpResponseBadRequest('Некорректно'
+                                          ' заполнены данные в форме')
+
+
+class StudentUpdateMain(View):
+    """
+    Redirect /student-update/ page to /students-info/
+    #:param request: output 'students-info.html'
+    #:return: prints /students-info/ page
+    """
+
+    def get(self, request): # noqa
+        return redirect(reverse('students_info'))
+
+
+class StudentUpdate(View):
+    """
+    This page updated information of students
+    #:param request:
+    #:param id: id of student in database
+    :return: updated students info page
+    """
+
+    def get_student(self, id): # noqa
+        return Student.objects.get(id=id) # noqa
+
+    def get(self, request, id): # noqa
+        student = self.get_student(id)  # noqa
+        one_student_form = StudentsAddForm(instance=student)
+        context = {'form': one_student_form, 'student_id': student.id,
+                   'student_name': student.name,
```

Comment:
better just send `'student': student` and in template you can all this methods
if you use this flow, each time when you want add new student data should add them to context, so just add to context student instance

Reply:
fixed